### PR TITLE
Documentation_update

### LINKED ISSUE #61: update documentation
Most documentation and other materials such as icons and links are still referencing onionshare. Currently someone hoping to build or use the project may become confused by the mixed references and lack of consistent mission statement or branding.

### LINKED ISSUE #78: Change all references to onionshare/micahflee in edited files
What's the comment for if the code is gone?
Also, Hyperdome*

_Originally posted by @makusu2 in https://github.com/arades79/hyperdome/pull/77#discussion_r407302319_

### PR DESCRIPTION
**Pull request by arades79**
### Summary

Update documentation for hyperdome 0.2 changes and add more description/demos.

Also changes nearly all references from onionshare to hyperdome

This PR fixes #61 and #78 
### Checklist

- [ ] If code changes were made then they have been tested.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -1,68 +1,67 @@
...
+
 Install Xcode from the Mac App Store. Once it's installed, run it for the first time to set it up. Also, run this to make sure command line tools are installed: `xcode-select --install`. And finally, open Xcode, go to Preferences > Locations, and make sure under Command Line Tools you select an installed version from the dropdown. (This is required for installing Qt5.)
 
-Download and install Python 3.7.2 from https://www.python.org/downloads/release/python-372/. I downloaded `python-3.7.2-macosx10.9.pkg`.
+Download and install Python 3.8.2 from https://www.python.org/downloads/release/python-382/.
 
-You may also need to run the command `/Applications/Python\ 3.7/Install\ Certificates.command` to update Python 3.6's internal certificate store. Otherwise, you may find that fetching the Tor Browser .dmg file fails later due to a certificate validation error.
+You may also need to run the command `/Applications/Python\ 3.8/Install\ Certificates.command` to update Python 3.6's internal certificate store. Otherwise, you may find that fetching the Tor Browser .dmg file fails later due to a certificate validation error.
```
Comment (stevenpitts) [inline BUILD.md]: to update Python 3.8's *

File: README.md [hunk 2]
```diff
@@ -16,7 +18,41 @@ The primary use case is mental health services. University campuses or businesse
...
+
+To add a server you intend to be a counselor for, click the counselor radio button, enter a username you don't mind the Server seeing, and your one-time counselor password.
+
+When you're ready to chat, just press *Start Chat*. Counselors will wait for a Guest to be paired with them, and Guests will connect to a random counselor. When you're done just hit *Disconnect*. Don't worry, all messages are encrypted not only by the Tor network, but by additional end-to-end encryption. Not to mention, all IPs are obscured by the onion protocol, and absolutely no chat metadata is stored. Hyperdome takes privacy and security very seriously!
+
+## Disclaimer
+Hyperdome is currently still pre-release software! At this time it is not reccomended to deploy a hyperdome server, or encourage it's wide use. Hyperdome has not been independently audited, and cannot be guaranteed to be safe. Also all interfaces are considered unstable, Server and client code may change drastically at any time. If you're still interested in demoing the software then feel free to build or run a release and do spread the word!
```
Comment (stevenpitts) [inline README.md]: encourage its*

**Review by stevenpitts — DISMISSED** [review]

File: BUILD.md (hunk 1 above)
Comment (arades79) [reply BUILD.md]: better: python's
and done

File: README.md (hunk 2 above)
Comment (arades79) [reply README.md]: done.

P.S. wow that was the only "it's" in the whole document and it was wrong

**Review by stevenpitts — APPROVED** [review]
Looks good!

**Pull request merged**